envs/dev/templates/team-geordi.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: team-geordi
+  namespace: argocd
+  finalizers:
+  - resources-finalizer.argocd.argoproj.io
+spec:
+  project: default
+  destination:
+    namespace: yelb
+    server: {{ .Values.spec.destination.server }}
+  source:
+    repoURL: {{ .Values.spec.source.repoURL }}
+    path: teams/team-geordi/dev
+    targetRevision: {{ .Values.spec.source.targetRevision }}
+    helm:
+      parameters:
+      - name: replicaCount
```

Comment:
If you are setting these options, then we need to provide them through the gitops bridge in the pattern. Are any of these significant/required?

Reply:
Yes i will be using the below in the pattern few significant ones :
```
            bootstrapValues: {
                service: {
                    type: 'LoadBalancer'
                },
                ingress: {
                    enabled: true,    
                    host: 'prod.blueprint.com',
                },
            },
```